AmazonChimeSDKDemo/AmazonChimeSDKDemo/AutomaticLanguageDetectionViewController.swift

```diff
@@ -0,0 +1,123 @@
+//
+//  AutomaticLanguageDetectionViewController.swift
+//  AmazonChimeSDKDemo
+//
+//  Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+//  SPDX-License-Identifier: Apache-2.0
+//
+
+import Foundation
+import AmazonChimeSDK
+import UIKit
+
+class AutomaticLanguageDetectionViewController: UIViewController, UITextFieldDelegate {
+    
+    @IBOutlet var languageDetectionTableView: UITableView!
+    @IBOutlet var preferredLanguagesTextField: UITextField!
+    var preferredLanguagesPickerView = UIPickerView()
+    public var cancellationHandler: ((Int?) -> Void)?
+    public var selectedLanguageOptionsHandler: ((String?) -> Void)?
+    public var preferredLanguageOptionHandler: ((String?) -> Void)?
+    public var languages: [String] = []
+    public var languagesDict: [String: String] = [:]
+    
+    var preferredLanguages = ["-- Optional --"]
+    var selectedLanguages = " "
+    var preferredLanguageSelected = " "
+
+    override func viewDidLoad() {
+        super.viewDidLoad()
+        self.preferredLanguagesPickerView.delegate = self
+        self.preferredLanguagesPickerView.dataSource = self
+        self.preferredLanguagesTextField.delegate = self
+        self.preferredLanguagesTextField.inputView = preferredLanguagesPickerView
+        preferredLanguagesTextField.text = preferredLanguages[0]
+    }
+    
+    func textField(_ textField: UITextField, shouldChangeCharactersIn range: NSRange, replacementString string: String) -> Bool {
+        return false
+    }
+    
+    @IBAction func saveLanguageDetectionOptions(_ sender: Any) {
+        if preferredLanguages.count < 3 {
```

Comment:
Rename `preferredLanguages` to more technically correct name `languageOptions`.

`preferredLanguage` is an optional parameter to be send to `start_transcription`. `languageOptions` is a required parameter along with `identifyLanguage` (boolean).

Refer - https://docs.aws.amazon.com/transcribe/latest/dg/API_streaming_StartStreamTranscription.html for naming conventions

Reply:
will be updated on next commit.